src/server/helpers/utils.js

```diff
@@ -52,13 +72,36 @@ function getEntityModelByType(type) {
 	return entityModels[type];
 }
 
+/**
+ * Regular expression for valid BookBrainz IDs
```

Comment:
"BookBrainz IDs" -> "UUIDs". It might also be worth renaming this to _uuidRegex (if you feel like it!), since it isn't specific to BBIDs (could work for MBIDs too)